database/src/migrations/20230719172000_project_type_clean_up.ts

```diff
@@ -0,0 +1,42 @@
+import { Knex } from 'knex';
+
+/**
+ * // add new table for region lookup and connection table to project and survey
```

Comment:
nit

Reply:
Also, wondering if this cleanup migration should be in the same migration file as the table creation, since they are 2 steps of the same overall process. 

Not sure if the grouped sql is better or if splitting it out is nicer. Thoughts?

Reply:
as per our discussion, moved those into a single migration